app/models/team.rb

```diff
@@ -2,6 +2,16 @@ class Team < ActiveRecord::Base
   after_create :create_tenant
   has_many :brews
   
+  def self.find_team(user_info)
```

Comment:
I think this method can be cleaned up a little bit more but it's fine for right meow.